galaxy.yml

```
@@ -39,8 +39,8 @@ tags:
 # L(specifiers,https://python-semanticversion.readthedocs.io/en/latest/#requirement-specification). Multiple version
 # range specifiers can be set and are separated by ','
 dependencies:
-  ansible.posix: 1.4.0
-  community.general: 1.3.6
+  ansible.posix: ">=1.4.0"
+  community.general: ">=1.3.6"
   adfinis.facts: 1.0.2
```

I'd do the same for the adfinis.facts collection as well:
```suggestion
  adfinis.facts: ">=1.0.2"
```